### DISCUSSION TITLE
Tab locations

### DISCUSSION THREAD

**Discussion by Jadhav-Sourabh** [pybamm-team/PyBaMM#2637] [General]
Can anyone give example about tab locations parameter for cylindrical cell?
    'Negative tab centre y-coordinate [m]': ,
    'Negative tab centre z-coordinate [m]': ,
    'Positive tab centre y-coordinate [m]': ,
    'Positive tab centre z-coordinate [m]':

**Comment by brosaplanella** [comment]
The tab locations apply for a pouch cell only and this feature is not supported for cylindrical cells.
